zc_install/sql/updates/mysql_upgrade_zencart_210.sql

```diff
@@ -34,7 +34,9 @@
 TRUNCATE TABLE whos_online;
 TRUNCATE TABLE db_cache;
 
-
+#PROGRESS_FEEDBACK:!TEXT=Adding error logging to email archive ...
+# Add column to store any errorinfo returned from phpmailer.
+ALTER TABLE ADD ALTER TABLE email_archive ADD COLUMN (errorinfo TEXT NULL);
```

Comment:
```suggestion
ALTER TABLE email_archive ADD COLUMN errorinfo TEXT DEFAULT NULL;
```

Reply:
Wow that was messy :) My lazy copy paste. Thanks.